tests/api2/test_ipa_join.py

```diff
@@ -85,7 +85,9 @@ def test_smb_keytab_exists(do_freeipa_connection):
 def test_admin_privilege(do_freeipa_connection, enable_ds_auth):
     ipa_config = call('ldap.ipa_config')
 
-    priv = call('privilege.query', [['name', '=', ipa_config['domain']]], {'get': True})
+    priv_names = [priv['name'] for priv in call('privilege.query')]
```

Comment:
priv is now undefined below

Also 2 flake8 indentation is not a multiple of 4 on lines 21 & 30 above